service/collector/app_operator.go

```diff
@@ -20,6 +20,7 @@ var (
 		prometheus.BuildFQName(namespace, "ready", "total"),
 		"Gauge with ready app-operator instances per app CR version.",
 		[]string{
+			labelNamespace,
```

Comment:
Sorry diff is a bit hard to follow. As both collectAppVersions and collectOperatorVersions need to become namespace aware.